solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -974,9 +975,9 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
      */
     function removeLiquidity(
```

Comment:
isn't this another interface breaking change?

Reply:
Yes.
It is solvable here, since we can overload the function and have the older version as part of the contract (unlike `function reserveTokens`, where the change is only in the return-value type).
However, it would be very unfortunate if we ended up keeping the older version. Those dynamic arrays yield a chaos in the contract.

Reply:
@yudilevi:
Yes, and this is indeed a problem in the LP contract which relies on the old versions of both `addLiquidity` and `removedLiquidity` (a problem which was "hidden from the compiler" by the `IConverter` interface that we have declared locally inside that contract).
In any case, I have overloaded these two functions with versions that take and return dynamic arrays.
They are placed at the bottom of the file (alongside all the other deprecated functions).
This will allow us to upgrade the LP contract independently of the upgrade of each one of the converters.
We should make sure that there are no other API-breaking changes, as this one has eluded the compiler.

Reply:
wouldn't that add yet another layer of friction for integrators? another public overloaded functions?

Reply:
No.
Onchain, the compiler knows which overloaded version to call, so there's no issue.
Offchain, the only friction is having to use the explicit (rather "ugly") syntax of web3. But this is an issue once you have 2 overloaded versions of the function, which is already the case before we add this (3rd) overloaded version.